clis/ph-cli/src/services/generate.ts

```diff
@@ -84,16 +92,20 @@ export async function startGenerate(
       appId: command.driveEditorAppId,
       allowedDocumentTypes: command.allowedDocumentTypes,
       isDragAndDropEnabled: command.isDragAndDropEnabled,
+      driveEditorDirName: command.driveEditorDirName,
     });
   } else if (command.editor) {
     if (!command.editorName) {
       throw new Error("Editor name is required (--editor or -e)");
     }
-    await generateEditor(
-      command.editorName,
-      command.documentTypes?.split(/[|,;]/g) ?? [],
-      config,
-    );
+    await generateEditor({
+      name: command.editorName,
```

Comment:
I've updated these to use an object arg because there were way too many similar args and they were getting mixed up in some places